src/components/social-links/SocialLinks.scss

```diff
@@ -0,0 +1,41 @@
+@import "../../styles/colors";
+
+#social-links {
```

Comment:
we are usually using classes for styling, not ids. With Ids you have to remember that it has higher specificity ( id, class, element )

Reply:
Try to rewrite it for classes (yeah - tags to ;) )

Reply:
that was my main idea - to encapsulate the styles within the component element only

Reply:
in other words - I wanted to achieve something like styled-components with pure SASS